internal/pfcpsim/server.go

```diff
@@ -0,0 +1,329 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2022 Open Networking Foundation
+ */
+
+package pfcpsim
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/c-robinson/iplib"
+	pb "github.com/omec-project/pfcpsim/api"
+	"github.com/omec-project/pfcpsim/pkg/pfcpsim"
+	"github.com/omec-project/pfcpsim/pkg/pfcpsim/session"
+	log "github.com/sirupsen/logrus"
+	ieLib "github.com/wmnsk/go-pfcp/ie"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	//FIXME: the SDF Filter is not spec-compliant. We should fix it once SD-Core supports the spec-compliant format.
+	defaultSDFfilter = "permit out ip from 0.0.0.0/0 to assigned 80-80"
```

Comment:
nit: it would be nice to have it passed as a cmd line argument in future

Reply:
Sure thing. Added a `TODO`